app/controllers/schedule_controller.rb

```diff
@@ -3,12 +3,6 @@
 class ScheduleController < ApplicationController
   def new
     @edition = Edition.find_current(document: params[:document])
-
-    unless @edition.schedulable?
-      # FIXME: this shouldn't be an exception but we've not worked out the
-      # right response - maybe bad request or a redirect with flash?
-      raise "Document is not in a schedulable state."
-    end
```

Comment:
I'd have thought this still needs something so users can't just access this via URL.

Reply:
Ah, my hope here was that we'd agree this wasn't particularly important, since it's just a GET action. If it helps, I don't think we have this kind of check on any other actions - just for modifier ones. Is there something we're particularly concerned about here?

Reply:
I'd imagine if you were to go to this route without a scheduled_publishing_datetime value you'd get an exception. Whereas I think if you went back to this form after scheduling with the back button you'd end up seeing the form again even if you're scheduled.

Most consistent approach we have is raising an exception when users access pages outside of the state they're supposed to be in. I think removing this will make https://trello.com/c/8PsWxNzY/651-audit-how-we-cope-with-duplicate-user-actions-eg-in-two-tabs harder